packages/cli/src/databases/dsl/Table.ts

```diff
@@ -116,3 +116,45 @@ export class DropColumns extends TableOperation {
 		return queryRunner.dropColumns(`${prefix}${tableName}`, columnNames);
 	}
 }
+
+export class AddNotNull extends TableOperation {
+	constructor(
+		prefix: string,
+		tableName: string,
+		protected columnName: string,
+		queryRunner: QueryRunner,
+	) {
+		super(tableName, prefix, queryRunner);
+	}
+
+	async execute(queryRunner: QueryRunner) {
+		const { tableName, prefix, columnName } = this;
+		const table = await queryRunner.getTable(`${prefix}${tableName}`);
+		if (!table) throw new Error(`No table found with the name ${tableName}`);
+		const oldColumn = table.findColumnByName(columnName)!;
+		const newColumn = oldColumn.clone();
+		newColumn.isNullable = false;
+		return queryRunner.changeColumn(table, oldColumn, newColumn);
+	}
+}
+
+export class DropNotNull extends TableOperation {
```

Comment:
tiny comment: Drop in the name suggests that the column is actually gone after this, I'd probably use RemoveNotNullAttribute or something clearer here?

Reply:
I'll leave this one to @netroy. I personally find it clear.